### DISCUSSION TITLE
How to edit expeditions?

### DISCUSSION THREAD

**Discussion by Graysonn** [zencq/NomNom#212] [Q&A]
I've completed the current expedition. I started it on my main save by choosing the option in the anomaly. Now I'm finished I thought I'd build some stuff to help other players. I wanted to unlock building parts. However whenever I load up the save it shows me the loadout from before I started the expedition. 

Is there anyway to edit the expedition items and knowledge?

**Comment by Wallopingaround** [comment]
When you edit raw JSON there is "Base Context" and "Expedition Context". So yeah. The latter is for an expedition.
